build.rs

```diff
@@ -13,7 +13,7 @@ fn styles() -> String {
 
 #[cfg(not(debug_assertions))]
 fn styles() -> String {
-    grass::include!("../public/style.css").to_string()
+    grass::include!("../public/style.scss").to_string()
```

Comment:
Shouldn't the path look like this? The build file is in the same directory as the `public` directory

```suggestion
    grass::include!("public/style.scss").to_string()
```

Reply:
ah shoot yeah